lib/shopify_cli/theme/theme_admin_api_throttler/bulk.rb

```diff
@@ -65,7 +66,7 @@ def consume_put_requests
               end
             end
           end
-          to_batch
+          [to_batch, to_batch_size_bytes]
```

Comment:
Nitpick, but can we remove the `to_batch_size_bytes` in favor of `to_batch.map(&:size).reduce(:+).to_i` (here and in the `BulkJob`)?

As we're memoizing `PutRequest#size`, it seems like calculating that wouldn't be expensive :)

Reply:
Let's leave it as is for now, following our Slack conversation, @jessetuglu  :)